packages/overlay/src/overlay-stack.ts

```diff
@@ -32,10 +32,67 @@ export class OverlayStack {
 
     public constructor() {
         this.addEventListeners();
+        this.initTabTrapping();
+    }
+
+    private tabTrapper!: HTMLElement;
+    private overlayHolder!: HTMLElement;
+
+    private initTabTrapping(): void {
+        this.document.body.attachShadow({ mode: 'open' });
```

Comment:
I was thinking about the body shadowRoot collision issue that we discussed offline.

What are your thoughts on cancelling the tabTrapping initiation if a shadowRoot already exists in the body?

Reply:
Not a bad idea for safety... document that users with other tools leveraging a `shadowRoot` on the `<body>` element just don't get the trapping functionality. Certainly better than just going and breaking apps that have this for some reason.